### PR TITLE
Adding support for getting all configmaps

### PR DESCRIPTION
**Pull request by SinaChavoshi**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2941)

### CODE REVIEW

File: sdk/python/kfp/cli/diagnose_me/kubernetes_cluster.py [hunk 1]
```diff
@@ -38,6 +39,7 @@ class Commands(enum.Enum):
     Commands.GET_SECRETS: 'get secrets',
     Commands.GET_SERVICES: 'get services',
     Commands.GET_KUBECTL_VERSION: 'version',
+    Commands.GET_CONFIG_MAP: 'get configmap',
```
Comment (Ark-kun) [inline sdk/python/kfp/cli/diagnose_me/kubernetes_cluster.py]: Which configmap? KFP will have several of them.

Comment (SinaChavoshi) [reply sdk/python/kfp/cli/diagnose_me/kubernetes_cluster.py]: good point, changing the enum to GET_CONFIG_MAPS ... this will return all configs maps.

**Pull request merged**